src/tape/ensemble_frame.py

```diff
@@ -178,6 +187,198 @@ def query(self, expr, **kwargs):
         result = super().query(expr, **kwargs)
         return self._propagate_metadata(result)
     
+    def merge(
```

Comment:
Is explicitly listing the kwargs here needed, or can they just be passed along via **kwargs to dask's merge call?

Reply:
Yeah this seems the correct call. Thanks!